WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -100,7 +100,7 @@ class PagesFragment : Fragment() {
     }
 
     private fun onPageEditFinished(pageId: Long) {
-        viewModel.onPageEditFinished()
+        viewModel.onPageEditFinished(pageId)
```

Comment:
> Wouldn't it be better to rely on the local post id here? 

Unfortunately that's not possible because the local ID changes every time it's updated. You can verify that by setting a breakpoint in the `PagesViewModel.onPostUploaded()` handler.

> it'd also make it impossible to decide which continuation to run if there are 2 OnPostUploaded events for local items

That's why we try to prevent simultaneous actions by hiding the action menu while another operation is in progress.

Reply:
> Unfortunately that's not possible because the local ID changes every time it's updated.

I couldn't understand why this was the case and put myself in a debug hole. This is due to us removing every non-local post in the DB each time the first page of the post list is fetched. Once my changes are merged in this won't be the case for blog posts, but I think it'll be like this for pages for a while.